app/travel_processor/deploy-both/ui5-deploy-node.yaml

```diff
@@ -0,0 +1,56 @@
+# yaml-language-server: $schema=https://sap.github.io/ui5-tooling/schema/ui5.yaml.json
+specVersion: "2.6"
+metadata:
+  name: sap.fe.cap.travelN
+type: application
+
+resources:
+  configuration:
+    propertiesFileSourceEncoding: UTF-8
+builder:
+  resources:
+    excludes:
+      - "/test/**"
+      - "/localService/**"
+  customTasks:
+    - name: ui5-task-zipper
+      afterTask: generateCachebusterInfo
+      configuration:
+        archiveName: travel-processor-node
+    - name: custBuildTask-manifest
+      beforeTask: generateManifestBundle
+      configuration:
+        stack: node
+    - name: custBuildTask-compPreload
+      afterTask: generateComponentPreload
+      configuration:
+        stack: node
+    - name: custBuildTask-xsApp
+      beforeTask: ui5-task-zipper
+      configuration:
+        stack: node
```

Comment:
I wonder if we could get away with this task.  Instead 
- have the union of all 3 routes in xs-app.json: `/processor`, `/processor-java`, `/processor-node`, pointing to `sflight-srv`, `sflight-java-srv`, and `sflight-node- srv`.
- In manifest.json, replace the route `/processor` accordingly.